integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/mortgage/MortgageSpark.scala

```diff
@@ -430,8 +430,17 @@ object Main {
       .appName("MortgageJob")
       .getOrCreate()
 
-    0.until(10).foreach { _ =>
-      Run.parquet(session, perfPath, acqPath).write.mode("overwrite").parquet(output)
+    // extend args to support csv/orc/parquet dataset
+    if (args.length > 5) {
+        throw new IllegalArgumentException("Supports up to 5 arguments.")
```

Comment:
I'm happy to see an error here, but the error is not very helpful. Ideally this should explain what arguments are expected.  Also for usage errors like this one tends not to throw an exception which prints a large stacktrace with it but instead just prints an error, e.g.:
```suggestion
        System.err.println("Usage: <perfpath> <acqpath> <outputpath> [csv|orc|parquet]")
        System.exit(1)

Reply:
Good suggestion, Updated